### ISSUE TITLE
unexpected regexp from nginx variable $time_iso8601

### ISSUE THREAD

**Issue by makhomed** [martin-helmich/prometheus-nginxlog-exporter#44]
**Issue type**:
Bug report

**What happened**:
nginx variable ```$time_iso8601``` converted to ```(?P<time_iso>[^8]*)8601``` regexp

**What you expected to happen**:
valid regexp expected, something like this: ```(?P<time_iso8601>)```

**How to reproduce it (as minimally and precisely as possible)**:
install nginx, add to nginx.conf:
```
log_format frontend '$time_iso8601\t$geoip_country_code\t$remote_addr\t$scheme\t$host\t$request_method\t'
                    '"$request_uri"\t$status\t$body_bytes_sent\t"$http_referer"\t"$http_user_agent"\t-\t'
                    '$request_time\t$upstream_response_time';

access_log /var/log/nginx/access.log frontend;
```

**Other important information**:
For details, see Alphabetical index of some nginx variables - http://nginx.org/en/docs/varindex.html
(but variables from 3-rd party modules not included in this list).

**Configuration file**:

```yaml
listen:
  port: 4040
  address: "172.22.22.101"

namespaces:
  - name: ht_router
    format: "$time_iso8601\t$geoip_country_code\t$remote_addr\t$scheme\t$host\t$request_method\t\"$request_uri\"\t$status\t$body_bytes_sent\t\"$http_referer\"\t\"$http_user_agent\"\t-\t$request_time\t$upstream_response_time"
    source_files:
      - /var/log/nginx/access.log
    relabel_configs:
      - target_label: host
        from: host
```

**Exporter output**:

```
error while parsing line '2018-07-19T23:21:56+03:00     RU      141.8.142.53    https   www.example.com    GET     "/robots.txt"   200     38      "-"     "Mozilla/5.0 (compatible; YandexBot/3.0; +http://yandex.com/bots)" -       0.000   0.001': access log line '2018-07-19T23:21:56+03:00      RU      141.8.142.53    https   www.example.com    GET     "/robots.txt"   200     38      "-"     "Mozilla/5.0 (compatible; YandexBot/3.0; +http://yandex.com/bots)" -       0.000   0.001' does not match given format '^(?P<time_iso>[^8]*)8601    (?P<geoip_country_code>[^       ]*)     (?P<remote_addr>[^      ]*)     (?P<scheme>[^   ]*)     (?P<host>[^        ]*)     (?P<request_method>[^   ]*)     "(?P<request_uri>[^"]*)"        (?P<status>[^   ]*)     (?P<body_bytes_sent>[^  ]*)     "(?P<http_referer>[^"]*)"       "(?P<http_user_agent>[^"]*)"    - (?P<request_time>[^      ]*)     (?P<upstream_response_time>[^ ]*)$'
```

**Example log file**:
(tabs converted to spaces, in real log file only one char \t is field separator)
```
2018-07-19T23:21:56+03:00     RU      141.8.142.53    https   www.example.com    GET     "/robots.txt"   200     38      "-"     "Mozilla/5.0 (compatible; YandexBot/3.0; +http://yandex.com/bots)" -       0.000   0.001': access log line '2018-07-19T23:21:56+03:00      RU      141.8.142.53    https   www.example.com    GET     "/robots.txt"   200     38      "-"     "Mozilla/5.0 (compatible; YandexBot/3.0; +http://yandex.com/bots)"     -       0.000   0.001
```

**Environment**:
- Exporter version: v1.2.0
- OS: CentOS Linux release 7.5
- Others: nginx/1.15.1 from official http://nginx.org/en/linux_packages.html#mainline repo

**Comment by martin-helmich** [comment]
Thanks for the report and sorry for the late response! This looks like an issue in the `github.com/satyrius/gonx` library used for parsing the log files. I'll see if I can raise this issue there, any maybe fix it directly.

**Comment by martin-helmich** [comment]
Luckily, this was already solved in satyrius/gonx#38. Fixed by updating to a recent version of that package.
